internal/sync/cache.go

```diff
@@ -70,15 +71,15 @@ type ResourceInfo struct {
 }
 
 // LiveStateCache is a cluster caching that stores resource references and ownership
-// references. It also stores custom metadata for resources managed by GitSyncs. It always
-// ensures the cache is up-to-date before returning the resources.
+// references. It also stores custom metadata for includedResources managed by GitSyncs. It always
+// ensures the cache is up-to-date before returning the includedResources.
```

Comment:
was this changed on purpose or by accident?

Reply:
yeah this is changed by IDE automatically, will revert it.